### PR TITLE
fix: Bad request on callback deletion

### PR DESCRIPTION
**Pull request by TheOptimisticFactory**
#### What is this PR doing?

1) Adding details of what when wrong when an `Unexpected response from OpenTok` occurred.
2) Fixing a bug where the method `unregisterCallback` would reject with a `400 - Bad Request` because the body was not null.

#### How should this be manually tested?

Steps:
1. create a callback using `registerCallback` 
2. then remove it using `unregisterCallback`

#### What are the relevant tickets?

- https://github.com/opentok/opentok-node/issues/290

### CODE REVIEW

File: lib/callbacks.js [hunk 1]
```diff
@@ -107,8 +107,9 @@ exports.unregisterCallback = function (config, callbackId, callback) {
     return;
   }
   api(
-    config, 'DELETE', '/callback/' + encodeURIComponent(callbackId), {},
-    function (err, response) {
+    config, 'DELETE', '/callback/' + encodeURIComponent(callbackId),
+    null,
```
Comment (TheOptimisticFactory) [inline lib/callbacks.js:111]: Having an object here will cause the request to reject with `400 - Bad request`

**Review by manchuck — APPROVED** [review]

**Comment by TheOptimisticFactory** [comment]
I am seeing the changes have been approved, what are the next steps?

The codecov part is confusing because it is states the coverage won't be altered while simultaneously indicating a diff percentage.
> # [Codecov](https://codecov.io/gh/opentok/opentok-node/pull/289?src=pr&el=h1&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=opentok) Report
> > Merging [#289](https://codecov.io/gh/opentok/opentok-node/pull/289?src=pr&el=desc&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=opentok) ([ca7097e](https://github.com/opentok/opentok-node/commit/ca7097e8e3e14d90cb23bb9e4dd7ffef58a28fd3)) into [main](https://codecov.io/gh/opentok/opentok-node/commit/aca5bda87be3a448344ae03d1c4fc857c8ef9926?el=desc&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=opentok) ([aca5bda](https://github.com/opentok/opentok-node/commit/aca5bda87be3a448344ae03d1c4fc857c8ef9926)) will **not change** coverage.
> > The diff coverage is `35.7%`.

**Review by manchuck — APPROVED** [review]

**Comment by manchuck** [comment]
@TheOptimisticFactory We will work on getting a release pushed out

**Comment by TheOptimisticFactory** [comment]
@manchuck Alright, thanks.

**Pull request merged**